### DISCUSSION TITLE
【灌水】寻找一位艺人

### DISCUSSION THREAD

**Discussion by qqqqqq77777** [share121/inter-knot#2721] [其他]
你们还记得几年前的“最美都市小姐”的比赛吗，虽然说是女生参加的比赛，可是我隐隐记得有一届有一位男艺人参加，表演的内容好像是赞美鸡的。当时觉得那个艺人挺帅的，可是后来落选了就销声匿迹了，有人知道他叫什么名字吗？
![image](https://github.com/user-attachments/assets/4ecd9ca2-8ef3-4d0f-9cca-c98dc1778898)
这是我当时拍的一张照，真的很想知道他是谁

**Comment by LZCCCCC518** [comment]
6年了

**Comment by hanhanhan12** [comment]
希腊奶，顶

**Comment by Ziew-Ryan** [comment]
His name is Cai Xvkun🙂

**Comment by ybbgh** [comment]
❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤

**Reply by LLLingYu** [reply]
点了啊，没有看到捏，是不是忘记放超链接了

**Reply by lllliufeng** [reply]
[❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤ ](https://www.bilibili.com/video/BV1im42137GG)

**Reply by ybbgh** [reply]
<a href="https://www.bilibili.com/video/BV17e8ceeEGq/?spm_id_from=333.788.recommend_more_video.5&vd_source=9f1c67940ef7fee251eca4e9f4089507" target="_blank">❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤</a>

**Comment by chihung3927** [comment]
啊↗啊↘啊→姬霓太美!蹬蹬!baby

**Comment by lllliufeng** [comment]
这不是我们鲲曲的创始人坤坤吗（

**Comment by yzysrewrwqr** [comment]
跟两年半前教我铁山靠的师傅有点像

**Comment by yivitte** [comment]
唯一一个宗主死了弟子还在的宗门
